packages/backend-core/src/objectStore/objectStore.ts

```diff
@@ -524,3 +525,26 @@ export async function getReadStream(
   }
   return client.getObject(params).createReadStream()
 }
+
+/*
+Given a signed url like 'files/signed/tmp-files-attachments/app_123456/myfile.txt' extract
```

Comment:
This does not really does what it says. The URL will start with `/`, but the documentation excludes it. Should we trim the initial `/` and accept both, or change the comment?

Reply:
Changed comment